app/Libs/ProductApiDriver.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: macintosh
+ * Date: 8/1/16
+ * Time: 1:20 PM
+ */
+
+namespace App\Libs;
+
+
+class ProductApiDriver
+{
+    public $apiUrl;
+    public $httpClient;
+
+    public function __construct() {
+        $this->apiUrl = env('CDS_PRODUCT_API_URL', '');
+        $this->httpClient = new \GuzzleHttp\Client([
+            'base_uri' => $this->apiUrl
+        ]);
+    }
+
+    private function getResponse($response) {
+        $content = $response->getBody()->getContents();
+        $resJson = json_decode($content);
+
+        return $resJson;
+    }
+
+    public function listAll($otherData, $page = 1) {
+        extract($otherData);
+        $orderQs = '';
+        if(isset($orderBy) && !empty($orderBy)) {
+            $orderQs = '&orderby='.$orderBy;
+            $orderQs .= '&order='.(isset($order) ? $order : 'desc');
+        };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1116)